docs/docs/developerguide/README.md

```diff
@@ -304,3 +318,44 @@ This package includes the script linked in the documentation above, which simpli
   DJANGO_GMAIL_API_CLIENT_SECRET='google_assigned_secret'
   DJANGO_GMAIL_API_REFRESH_TOKEN='google_assigned_token'
   ```
+
+
+#### Teamware Privacy Policy and Terms & Conditions
+
+Teamware includes a default privacy policy and terms & conditions, which are required for running the application.
+
+The default privacy policy is intended to be compliant with UK GDPR regulations, which may comply with the rights of users of your deployment, however it is your responsibility to ensure that this is the case.
+
+If the default privacy policy covers your use case, then you will need to include configuration for a few contact details.
+
+Contact details are required for the **host** and the **administrator**: the **host** is the organisation or individual responsible for managing the deployment of the teamware instance and the **administrator** is the organisation or individual responsible for managing users, projects and data on the instance. In many cases these roles will be filled by the same organisation or individual, so in this case specifying just the **host** details is sufficient.
+
+For deployment from source, set the following environment variables:
+
+* `PP_HOST_NAME`
+* `PP_HOST_ADDRESS`
+* `PP_HOST_CONTACT`
+* `PP_ADMIN_NAME`
+* `PP_ADMIN_ADDRESS`
+* `PP_ADMIN_CONTACT`
+
+For deployment using docker-compose, set these values in `.env`.
+
+##### Including a custom Privacy Policy and/or Terms & Conditions
+
+If the default privacy policy or terms & conditions do not cover your use case, you can easily replace these with your own documents.
+
+If deploying from source, include HTML files in a `custom-policies` directory in the project root with the exact names `custom-policies/privacy-policy.html` and/or `custom-policies/terms-and-conditions.html` which will be rendered at the corresponding pages on the running web app.
```

Comment:
We should make clear that these files are HTML _fragments_ to be inserted inside a `div`, not complete HTML pages with `head` and `body`.  Or we have to parse them somehow in Python and extract just the body content.

Or would it be too much of a stretch to allow for Markdown as an alternative? Use `custom-policies/privacy-policy.html` verbatim if it exists, `<MarkdownRenderer :content={custompolicies/privacy-policy.md}>` if not.

Reply:
That might be the cleanest option. Markdown allows (limited) free form HTML anyway if they want to go that route.

Reply:
Done in [90c9425](https://github.com/GateNLP/gate-teamware/pull/298/commits/90c9425b203ca67e31ff1fc9f31433b8aabeacf9)